### PR TITLE
fix link to recently-moved MoveIt for ROS 1 tutorials

### PR DESCRIPTION
**Pull request by v4hn**
As noted here: https://discourse.ros.org/t/move-of-nav2-and-moveit-repositories-at-github/37450/21?u=v4hn

### CODE REVIEW

**Comment by v4hn** [comment]
My immediate concern was to get the MoveIt ROS one reference back working for users, not to fix the migration for the whole repository, with CI broken for different unrelated reasons at the moment.

But fair enough. I went through the repo with a few `sed`s and changed all links and references.

One thing that popped up is that the github container registry is not migrated (yet?), e.g., `ghcr.io/ros-planning/moveit2_tutorials`. @henningkayser
I added the last commit to change those links for reference here, but it should not be merged yet.
Feel free to just drop that commit again if you are ok with merging the rest.

**Review by rhaschke — APPROVED** [review]
Thanks!

File: index.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ MoveIt 2 Documentation
 
 Welcome to the unified MoveIt documentation, which includes tutorials, how-to-guides, core concepts, and more.
 
-MoveIt 2 is the robotic manipulation platform for ROS 2, and incorporates the latest advances in motion planning, manipulation, 3D perception, kinematics, control, and navigation. MoveIt 2 was first released in 2019; for ROS 1 documentation, see `MoveIt 1 tutorials <https://ros-planning.github.io/moveit_tutorials>`_. For the commercially supported version see `MoveIt Pro tutorials <https://docs.picknik.ai/en/stable/>`_.
+MoveIt 2 is the robotic manipulation platform for ROS 2, and incorporates the latest advances in motion planning, manipulation, 3D perception, kinematics, control, and navigation. MoveIt 2 was first released in 2019; for ROS 1 documentation, see `MoveIt 1 tutorials <https://moveit.github.io/moveit_tutorials>`_.  For the commercially supported version see `MoveIt Pro tutorials <https://docs.picknik.ai/en/stable/>`_.
```
Comment (sjahr) [inline index.rst:6]:
```suggestion
MoveIt 2 is the robotic manipulation platform for ROS 2, and incorporates the latest advances in motion planning, manipulation, 3D perception, kinematics, control, and navigation. MoveIt 2 was first released in 2019; for ROS 1 documentation, see `MoveIt 1 tutorials <https://moveit.github.io/moveit_tutorials>`_. For the commercially supported version see `MoveIt Pro tutorials <https://docs.picknik.ai/en/stable/>`_.
```

**Review by henningkayser — APPROVED** [review]

**Comment by henningkayser** [comment]
CI is currently failing because of ros2_kortex, which needs https://github.com/Kinovarobotics/ros2_kortex/pull/159. I'll push a branch to generate new docker images, the registry is working as intended.

**Pull request merged**